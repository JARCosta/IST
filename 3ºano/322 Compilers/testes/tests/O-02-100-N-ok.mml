auto f = (double y) -> double { return y; };
begin
  f(10)!!
  f(2)!!
  return 0;
end

