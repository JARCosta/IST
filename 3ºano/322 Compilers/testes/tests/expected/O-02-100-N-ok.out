1E1
2
